<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgTagDialog</class>
 <widget class="QDialog" name="HgTagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Tag</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="tagCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Enter the name of the tag</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Tag Name&lt;/b&gt;
&lt;p&gt;Enter the name of the tag to be created, moved or deleted.&lt;/p&gt;</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Revision:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="revisionEdit">
     <property name="toolTip">
      <string>Enter a revision to set a tag for</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="tagActionGroup">
     <property name="title">
      <string>Tag Action</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="createTagButton">
        <property name="toolTip">
         <string>Select to create a tag</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Create Tag&lt;/b&gt;
&lt;p&gt;Select this entry in order to create a tag.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Create Tag</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="deleteTagButton">
        <property name="toolTip">
         <string>Select to delete a tag</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Delete Tag&lt;/b&gt;
&lt;p&gt;Select this entry in order to delete the selected tag.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Delete Tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="tagTypeGroup">
     <property name="title">
      <string>Tag Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="globalTagButton">
        <property name="toolTip">
         <string>Select to create/delete a global tag</string>
        </property>
        <property name="text">
         <string>Global Tag</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="localTagButton">
        <property name="toolTip">
         <string>Select to create/delete a local tag</string>
        </property>
        <property name="text">
         <string>Local Tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>tagCombo</tabstop>
  <tabstop>revisionEdit</tabstop>
  <tabstop>createTagButton</tabstop>
  <tabstop>deleteTagButton</tabstop>
  <tabstop>globalTagButton</tabstop>
  <tabstop>localTagButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgTagDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgTagDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteTagButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>revisionEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
